api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -77,4 +79,26 @@ public boolean isUsernameTaken(String username) {
       throw new RuntimeException(e);
     }
   }
+
+  public void createUser(String givenName, String familyName, String username, String password) {
+    String gSuiteDomain = configProvider.get().googleDirectoryService.gSuiteDomain;
+    User user = new User()
+      .setPrimaryEmail(username+"@"+gSuiteDomain)
+      .setPassword(password)
+      .setName(new UserName().setGivenName(givenName).setFamilyName(familyName));
+    try {
+      getGoogleDirectoryService().users().insert(user).execute();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public void deleteUser(String username) {
+    String gSuiteDomain = configProvider.get().googleDirectoryService.gSuiteDomain;
+    try {
+      getGoogleDirectoryService().users().delete(username+"@"+gSuiteDomain).execute();
```

Comment:
Do you know if this deletes the account permanently? Can you reuse the username once it's been deleted?

I'm not sure yet what our desired behavior is going to be for deleting users.... we'll need to think about that.

Reply:
You can reuse it. The integration test uses the same username each time it runs.